lib/recurly/apple-pay.js

```diff
@@ -191,6 +186,36 @@ class ApplePay extends Emitter {
     });
   }
 
+  /**
+   * Configures the methodology for pricing
+   *
+   * If a Pricing or PricingPromise instance is available, it will be used. Otherwise, a total
+   * will be expected.
+   *
+   * @param  {Object} options
```

Comment:
Would probably be good to document the properties here

Reply:
They’re documented in the constructor, so just didn’t want to repeat myself here.